### ISSUE TITLE
[Topic] Design patterns to avoid: Toasts

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#216]
To be created:
URL: https://wpaccessibility.org/docs/topics/design/avoid/toasts 
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/topics/design/avoid/toasts

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

This content should be written.

Resources: 
- [GitHub no longer uses toasts because of their accessibility and usability issues.](https://primer.style/accessibility/toasts/).
- [Defining ‘Toast’ Messages](https://adrianroselli.com/2020/01/defining-toast-messages.html)
